src/components/__tests_/Africa.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Africa from '../Africa';
+
+it('Renders air quality index header', () => {
+  render(<Africa />);
+  const aqiHeader = screen.getByText(/ Air Quality Index/i);
+  expect(aqiHeader).toBeInTheDocument();
+});
+
+it('Snapshot test for africa component passes  ', () => {
+  const tree = renderer.create(<Africa />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly work on the test that is failing in this file.

![image](https://user-images.githubusercontent.com/50941074/194719751-fd3b86af-ad1c-43f1-94e5-77b847cfa660.png)